util/util.go

```diff
@@ -4,10 +4,17 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/google/uuid"
-	"github.com/gorilla/mux"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+)
+
+// Permission names
+const (
+	VIEW_ADMIN   = "viewAdmin"
+	MODIFY_ADMIN = "modifyAdmin"
 )
```

Comment:
Great!
In JS I usually create an object with keys. In this case it would be something like
```
const PERMISSIONS = {
  VIEW_ADMIN: "viewAdmin",
  MODIFY_ADMIN:"modifyAdmin"
}
```
In that way, you can use them as `PERMISSIONS.VIEW_ADMIN`. It might be too long, but it makes it very clear that this is a permission.

Reply:
This also makes me think of enums in Rust. I'm not sure how go enums work though, if that could be applicable here.

Reply:
From what I can see, this seems to be the idomatic way of doing enums in go

Reply:
It would probably be good to move theme to the `auth` rather than util

Reply:
> 

Let's use this way then.
And moving them to auth makes more sense :+1: